pkg/operator/operator/k8s.go

```diff
@@ -388,6 +388,10 @@ func getEnvVars(api *spec.API, container string) []kcore.EnvVar {
 				Name:  "CORTEX_PROJECT_DIR",
 				Value: path.Join(_emptyDirMountPath, "project"),
 			},
+			kcore.EnvVar{
```

Comment:
Should we consider baking it into the container image?

Reply:
I agree with David. I think this env can be baked into the container image.